crates/config/src/toml.rs

```diff
@@ -551,8 +551,48 @@ where
 mod tests {
     use super::*;
 
+    fn build_plugins() -> Result<(), Box<dyn std::error::Error>> {
+        let root = Path::new("../..");
+        let src_file = root.join(file!());
+        let base = src_file.parent().unwrap_or(Path::new("."));
+
+        if !base
+            .join("../../../tests/dist/plugins/bulwark_blank_slate.wasm")
+            .exists()
+        {
+            bulwark_build::build_plugin(
+                base.join("../../../crates/wasm-sdk/examples/blank-slate"),
+                base.join("../../../tests/dist/plugins/bulwark_blank_slate.wasm"),
+                &[],
+                true,
+            )?;
+            assert!(base
+                .join("../../../tests/dist/plugins/bulwark_blank_slate.wasm")
+                .exists());
+        }
+
+        if !base
+            .join("../../../tests/dist/plugins/bulwark_evil_bit.wasm")
+            .exists()
+        {
+            bulwark_build::build_plugin(
+                base.join("../../../crates/wasm-sdk/examples/evil-bit"),
+                base.join("../../../tests/dist/plugins/bulwark_evil_bit.wasm"),
+                &[],
+                true,
+            )?;
+            assert!(base
+                .join("../../../tests/dist/plugins/bulwark_evil_bit.wasm")
+                .exists());
+        }
+
+        Ok(())
+    }
```

Comment:
The build function short-circuits if the files already exist, so it doesn't create unnecessary builds. In this context, the files need only exist, they won't be run or otherwise used.

Reply:
The relative file paths here do seem messy. I generally prefer to specify filenames relative to the current source file and that's what I was trying to do here, but it ended up being harder to read than I'd like. I'm not sure what the preferred way to do this would be if what I have here isn't it.

Reply:
I ended up fixing this by using `CARGO_MANIFEST_DIR` instead.